application/operation.py

```diff
@@ -29,6 +29,7 @@ class Option:
 	SILENT = 5
 	SILENT_MERGE = 6
 	SILENT_OVERWRITE = 7
+	FOLLOW_TO_SYMLINK = 8
```

Comment:
Rename to `FOLLOW_SYMLINK` as symlink is on source, following _to_ implies destination. :)